tests/test_spawner.py

```diff
@@ -589,8 +592,11 @@ def test_clustername(*args, **kwargs):
 
     config_built = spawner._build_cluster_config()
 
-    subnet_region = config_built['config']['gce_cluster_config']['subnetwork_uri'].split('/')[-3]
-    assert subnet_region == spawner.region
+    assert config_built['config']['gce_cluster_config']['subnetwork_uri'].split('/')[-3] == spawner.region
+    assert config_built['config']['master_config']['machine_type_uri'].split('/')[-3] == spawner.zone
```

Comment:
This seems to be checking the wrong thing: shouldn't we assert that the machine type URI zone matches the zone the user selection to spawn the cluster in?

Reply:
Good catch. Update in https://github.com/GoogleCloudDataproc/jupyterhub-dataprocspawner/pull/14/commits/d017eee3849e39413928b1b4a5ae2e124537f836